docs/source/SOFTWARE_INFRASTRUCTURE.rst

```diff
@@ -0,0 +1,62 @@
+.. _SOFTWARE_INFRASTRUCTURE:
```

Comment:
This file currently seems to add very little.

Reply:
Right now I agree it doesn't add much. Ideally I think it could be helpful for orienting new users, but I need help figuring out the best information to add. I'm fine with holding off and moving it into a new pull request if you'd prefer.